demo/lib/vm.bash

```diff
@@ -365,12 +365,12 @@ vm-pipe-to-file() { # script API
     #   echo 'Ahoy, Matey...' | vm-pipe-to-file /etc/motd
     local tmp append
     tmp="$(mktemp vm-pipe-to-file.XXXXXX)"
-    if [ "$1" = "--append" ]; then
+    if [ "$1" == "--append" ]; then
```

Comment:
Why this change to double equal signs?

Reply:
That seems to be a bad habit - even man page says that `=` is to be used with `[`. Thanks!

Reply:
Fixed

Reply:
I think both work in bash. But single equal sign is the POSIX standard or smth